metadata.go

```diff
@@ -240,6 +262,13 @@ type schemaDescriber struct {
 	cache map[string]*KeyspaceMetadata
 }
 
+// queries the cluster for schema information for a virtual keyspace
+type virtualSchemaDescriber struct {
+	session *Session
+	mu      sync.Mutex
+	cache   map[string]*VirtualKeyspaceMetadata
```

Comment:
I would recommend, in this particular case, use `sync.Map` instead of Mutex + map.

Reply:
I think it is preferable to use mutex, to keep the existing pattern, and avoid unnecessary code complexity.